tests/foundry/test/Token.t.sol

```diff
@@ -8,7 +8,7 @@ contract TokenTest {
         assert(true);
     }
 
-    //function test_2() public pure {
-    //    assert(false);
-    //}
+    function test_2() public pure {
```

Comment:
```suggestion
    function testFail_2() public pure {
```
Should we be ensuring that `forge tests` succeeds?

Reply:
No, then we would never be testing that our tool fails correctly on some inputs. We need such "negative" tests too, tests where we say `test_...` but it actually fails, or we say `testFail...` but it actually passes.